packages/eddsa-pcd/src/EdDSAPCD.ts

```diff
@@ -175,11 +176,15 @@ export const EdDSAPCDPackage: PCDPackage<
 };
 
 export async function getEdDSAPublicKey(
-  privateKey: string
+  privateKey: string | Uint8Array
 ): Promise<EDdSAPublicKey> {
   await ensureInitialized();
 
+  if (typeof privateKey === "string") {
+    privateKey = fromHexString(privateKey);
+  }
+
   return eddsa
-    .prv2pub(fromHexString(privateKey))
-    .map(toHexString) as EDdSAPublicKey;
+    .prv2pub(privateKey)
+    .map((p) => (eddsa as any).F.toObject(p).toString(16)) as EDdSAPublicKey;
```

Comment:
we probably need to change the type of `EDdSAPublicKey` to not be `[string, string]`

Reply:
What type should it have?

Reply:
Actually never mind - I misunderstood what `normal` form means - it's still a tuple of two values